image_assets/tests/mixins.py

```diff
@@ -46,6 +47,12 @@ def create_uploaded_image(cls,
         return SimpleUploadedFile(filename, buffer.getvalue(),
                                   content_type=content_type)
 
+    @classmethod
+    def create_uploaded_video(cls, filename="video.mp4") -> SimpleUploadedFile:
+        with open(Path(__file__).parent / "data/mark.mp4", 'rb') as f:
+            return SimpleUploadedFile(filename, f.read(10000),
```

Comment:
For an mp4 file reading file head is not enough, because all metadata may be located at the end of file

Reply:
`mark.mp4` is 3kb, why partial read?